doc/api/deprecations.md

```diff
@@ -660,6 +660,28 @@ Type: Runtime
 
 `REPLServer.parseREPLKeyword()` was removed from userland visibility.
 
+<a id="DEP00XX"></a>
+### DEP00XX: tls.parseCertString()
+
+Type: Runtime
+
+`tls.parseCertString()` is a trivial parsing helper that was made public by
+mistake. This function can usually be replaced with
+
+```js
+const querystring = require('querystring');
+querystring.parse(str, '\n', '=');
+```
+
+*Note*: This function is not completely equivalent to `querystring.parse()`, one
+notable difference is that `querystring.parse()` does url encoding, e.g.:
```

Comment:
Did you mean decoding?

Reply:
Urgs, somehow this comment slipped through. I still had the issue open as "should land soon".